server/app/assets/javascripts/applicant_entry_point.ts

```diff
@@ -16,6 +16,15 @@ import * as azureUpload from './azure_upload'
 import * as phoneNumber from './phone'
 import * as apiDocs from './api_docs'
 import * as trustedIntermediary from './trusted_intermediary'
+import * as htmx from './htmx'
+
+declare global {
+  interface Window {
+    htmx: typeof htmx
+  }
+}
+
+window.htmx = htmx
```

Comment:
```suggestion
import htmx from './htmx'
```

(blocking) When I upgraded htmx to v2 a month or two ago I had to I had to jump through a few hoops in order for webpack to correctly build.

You should just need to reference it this way for htmx to correctly get injected by webpack.